apps/personalization/models.py

```
@@ -0,0 +1,214 @@
+        default=0,
+        help_text="Total duration listened in this session (milliseconds)",
+    )
+
+    played_at = models.DateTimeField(
+        help_text="When the playback session occurred",
+    )
+
+    class Meta:
+        verbose_name = "Listening History"
+        verbose_name_plural = "Listening History"
+        unique_together = [["user", "asset", "surah_number"]]
```

Ruff (0.15.4)

[warning] Mutable default value for class attribute

(RUF012)